docs/excel/custom-functions-localize.md

```diff
@@ -0,0 +1,54 @@
+---
+ms.date: 03/15/2019
+description: Localize your Excel custom functions. 
+title: Localization of Excel custom functions (preview)
+localization_priority: Normal
+---
+# Localize custom functions
+
+Just like add-ins, custom functions can be localized. You'll provide locale information in the XML manifest file of your add-in and provide your own localized function names in the functions' JSON file.
+
+## Localize your add-in
+
+Before localizing your function names, you'll need to add optional locales to your XML file. For more information on this process, see [Localization for Office Add-ins](../develop/localization.md#control-localization-from-the-manifest).
```

Comment:
Would it be possible to show the locale for the German example below for manifest.xml?

Reply:
Yep